release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1675,6 +1677,8 @@ FATAL failed to fetch Metadata: failed to load asset "Install Config": platform.
 
 * Previously, and for unknown reasons, a kubelet could register the wrong IP address for a node. As a consequence, the node would be in a `NotReady` state until it was rebooted. Now, the systemd manager configuration is reloaded with the valid IP address as an environment variable, meaning that nodes no longer enter a `NotReady` state because a kubelet registered the wrong IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940939[*BZ#1940939*])
 
+* Previously, refactoring for a shadowed variable caused a regression related to the use of the checkpoint file, and SR-IOV pod sandboxes would not start up. A check for the path of the kubelet socket was not properly accounted for during the refactor. The fix properly restores the check for the kubelet socket path, and now the SR-IOV pod sandboxes are properly created. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1968625[*BZ#1968625*])
```

Comment:
> 🚫  start up verb
Do not use. Use "start"

Reply:
That's a very helpful catch, thank you! I appreciate the quick review.